git-link.el

```diff
@@ -471,6 +482,15 @@ return (FILENAME . REVISION) otherwise nil."
 
       (list host path))))
 
+(defun git-link--ssh-resolve-hostname (hostname)
+  "Resolve HOSTNAME using ssh client."
+  (let ((output (shell-command-to-string (format "ssh -G %s" hostname)))
+	(host nil))
+    (dolist (line (split-string output "\n"))
+      (when (cond ((string-match "^hostname \\(.*\\)" line) (match-string 1 line)))
```

Comment:
Do we need `cond`? If `(string-match "^hostname \\(.*\\)" line)` returns `t` then we know `(match-string 1 line)` will be non-`nil`, right?

Reply:
You're right, it's not needed. I have simplified it based on your feedback. Thanks!